### ISSUE TITLE
Alt text for "Powered by Esri" logo

### ISSUE THREAD

**Issue by jmccarth** [Esri/esri-leaflet#490]
The "Powered by esri" logo in the bottom right has no alt text, which causes accessibility checker errors. It's easy enough to hack the text in with jQuery after the map loads with something like this: 

`$('.esri-leaflet-logo img').attr('alt','Powered by Esri logo');`

but it would be nice if it just happened out of the box. That would provide an added benefit of consistent alt text across sites.

Probably not the biggest issue in the world, but hopefully a simple fix.

**Comment by jgravois** [comment]
good catch!

we declare the HTML for the control [here](https://github.com/Esri/esri-leaflet/blob/master/src/Controls/Logo.js#L15).  i'll fix soon unless you'd like to submit a PR.

**Comment by jmccarth** [comment]
Looks like you've got it taken care of already, thanks for the quick response!
